third_party/move/move-vm/integration-tests/src/tests/regression_tests.rs

```diff
@@ -79,7 +79,7 @@ fn script_large_ty() {
 
     let script_code = &mut decompiled_script.code.code;
     script_code.clear();
-    let num_vecpacks = 1000;
+    let num_vecpacks = 1;
```

Comment:
why the adjustment?

Reply:
It's because
1. This test only needs 1 vec pack to test the per-type limit
2. Having many vec packs with a large type will trigger the new metering mechanism, raising a different error code